Assets/Scripts/ScriptingSystem/GameManager.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+    private class nodeHolder
+    {
+        Head head;
```

Comment:
No encapsulation defined, it's not automatically set to private, please define encapsulation